lib/constants/routes.dart

```diff
@@ -121,6 +121,42 @@ class Routes {
     name: 'completedEntrepreneurSignup',
     path: '${signup.path}/completedEntrepreneurSignup',
   );
+  static Route signupMentorHelpSelection = Route(
+    name: 'signupMentorHelpSelection',
+    path: '${signup.path}/signupMentorHelpSelection',
+  );
+  static Route signupMentorMoreInfo = Route(
+    name: 'signupMentorMoreInfo',
+    path: '${signup.path}/signupMentorMoreInfo',
+  );
+  static Route signupMentorProfilePic = Route(
+    name: 'signupMentorProfilePic',
+    path: '${signup.path}/signupMentorProfilePic',
+  );
+  static Route signupMentorPronouns = Route(
+    name: 'signupMentorPronouns',
+    path: '${signup.path}/signupMentorPronouns',
+  );
```

Comment:
These three pages are pretty much the same as the ones for entrepreneur. It is a waste to recreate them, instead we should reuse the existing ones.

Determining which page goes next and whether to display  "Find mentors" or "Find entrepreneurs" in the MoreInfo left button can be done by inspecting the current user to determine if they are a mentor or an entrepreneur.

I am aware that we still don't have the logic to pass data between these pages, but I am just letting you know that we should not spend any time creating duplicate pages like these.